packages/contracts/test/util/fixtures.ts

```diff
@@ -49,3 +55,33 @@ export async function setupConfigWitMinterFilterV2Suite() {
   await config.minterFilter.approveMinterGlobally(config.minter.address);
   return config;
 }
+
+export async function setupSplits() {
+  const config = await loadFixture(setupConfig);
+  // deploy splitter implementation
+  config.splitterImplementation = await deployAndGet(
+    config,
+    "SplitAtomicV0",
+    []
+  );
+  // deploy splitter factory
+  config.splitterFactory = await deployAndGet(config, "SplitAtomicFactoryV0", [
+    config.splitterImplementation.address,
+    config.accounts.deployer.address, // required split address
+    2222, // required split bps
+  ]);
+  // deploy splitter via factory
+  const tx = await config.splitterFactory.createSplit([
+    { recipient: config.accounts.deployer.address, basisPoints: 2222 },
+    { recipient: config.accounts.artist.address, basisPoints: 2778 },
+    { recipient: config.accounts.additional.address, basisPoints: 5000 },
+  ]);
+  const receipt = await ethers.provider.getTransactionReceipt(tx.hash);
+  // get splitter address from logs
+  const splitterCreationLog = receipt.logs[receipt.logs.length - 1];
+  const splitterAddress = ethers.utils.getAddress(
+    "0x" + splitterCreationLog.topics[1].slice(-40)
+  );
```

Comment:
@Jbern16 here is an example of how to create a new splitter contract, as well as get the newly created address from the emitted event in the transaction